contracts/contracts/coordination/FlatRateFeeModel.sol

```diff
@@ -53,11 +53,29 @@ contract FlatRateFeeModel is IFeeModel, Ownable {
         uint256 numberOfProviders,
         uint32 duration
     ) external override {
+        processPayment(initiator, ritualId, numberOfProviders, duration);
+    }
+
+    function processRitualExtending(
+        address initiator,
+        uint32 ritualId,
+        uint256 numberOfProviders,
+        uint32 duration
+    ) external override {
+        processPayment(initiator, ritualId, numberOfProviders, duration);
+    }
```

Comment:
If the code is the same than `processRitualPayment()`, do you think we need a separate function?

Reply:
yes, `FlatRateFeeModel` is just one fee model, for subscription is different